index.html

```diff
@@ -167,23 +171,24 @@ <h2 class="about-title">About me</h2>
     </section>
   </main>
   <footer>
-    <section class="footer-head">
-        <h3>I'm always <br> interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
-    </section>
+    <section class="form-container">
+      <section class="footer-head">
+      <h3>I'm always <br class="h3-break"> interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+      </section>
 
-    <form class="footer-form" action="https://formspree.io/f/xayvwlqd" method="post">
-        <input type="text" id="name" name="user-name" placeholder="Full Name" required maxlength="30">
-        <input type="email" id="email" name="user-email" placeholder="Email Address" required>
-        <textarea name="message" id="msg" maxlength="500" placeholder="Enter Text Here" required></textarea>
-        <button id="form-button" type="submit">Get in Touch</button>
-    </form>
+      <form class="footer-form" action="https://formspree.io/f/xayvwlqd" method="post">
+          <input type="text" id="name" name="user-name" placeholder="Full Name" required maxlength="30">
+          <input type="email" id="email" name="user-email" placeholder="Email Address" required>
+          <textarea name="message" id="msg" maxlength="500" placeholder="Enter Text Here" required></textarea>
+          <button id="form-button" type="submit">Get in Touch</button>
+      </form>
+    </section>
 
     <hr class="footer-social-network">
 
     <ul class="footer-Links">
         <li><a href="#"><img src="assets/github-dark.png" alt="github" ></a></li>
-        <li><a href="#"><img src="assets/linkedin-dark.png"
-                    alt="linkedin"></a></li>
+        <li><a href="#"><img src="assets/linkedin-dark.png" alt="linkedin"></a></li>
         <li><a href="#"><img src="assets/hand-dark.png" alt="idk"></a></li>
         <li><a href="#"><img src="assets/twitter-dark.png" alt="twitter"></a></li>
         <li><a href="#"><img src="assets/m-dark.png" alt="M"></a></li>
```

Comment:
- [ ] [OPTIONAL] You've designed this entire page very well 👏. I want you to notice that the same separator at the bottom (`.footer-links`). that separator is supposed to span the entire page on desktop (or on all screen sizes). The previous reviewer didn't mention it so I've made it optional for you. please go ahead and make this change.
## Your Design 
![image](https://user-images.githubusercontent.com/18730106/144501138-5e609835-bd0c-41a5-92ad-5e692854da7e.png)

## Figma Design
![image](https://user-images.githubusercontent.com/18730106/144501503-1fae83a1-5838-4eed-b765-e4cd3c58bb86.png)

Reply:
Thanks! I'll do it 👍🏻